### PR TITLE
Option to disable front page video

### PR DESCRIPTION
**Pull request by MergHQ**
Someone requested this.

The video player is simply replaced with this:
80ee98327932f69a35c25d209292ef74

### CODE REVIEW

File: src/settings-list.js [hunk 1]
```diff
@@ -272,6 +272,25 @@ module.exports = [
         storageKey: 'disableWhispers'
     },
     {
+        name: 'Disable Frontpage Video',
+        description: 'Disable video on the frontpage',
+        default: false,
+        storageKey: 'disableFPVideo',
+        load: function() {
+            if (window.location.href === 'https://www.twitch.tv/' && bttv.settings.get('disableFPVideo') === true) {
+                $(window).load(function() {
```
Comment (night) [inline src/settings-list.js]: This is ran after the page loads: https://github.com/night/BetterTTV/blob/master/src/main.js#L311

**Comment by night** [comment]
I would much prefer you auto-pause the existing player rather than remove it.

**Comment by MergHQ** [comment]
So the video player would not be removed at all, just auto paused?

**Comment by night** [comment]
That's the idea. It's a really bad idea to remove it entirely, as some users might want to still preview the featured stream.

**Comment by MergHQ** [comment]
Okay, I made the changes. It's now paused instead of removed completely.

**Comment by night** [comment]
You did not address my code comment from earlier: https://github.com/night/BetterTTV/pull/1119#discussion_r62616373

**Comment by MergHQ** [comment]
The problem is that the DOM is loaded before the video iframe. Look like something the site dynamically creates the iframe when the window is ready.

At least i can't get it to work if i remove $(window).load

**Pull request merged**